library/IvozProvider/Klear/Filter/OutgoingDDI.php

```diff
@@ -10,8 +10,13 @@ public function setRouteDispatcher(KlearMatrix_Model_RouteDispatcher $routeDispa
         $pk = $routeDispatcher->getParam("pk", false);
 
         // Only display DDIs belonging to edited company
-        $this->_condition[] = "`companyId` = '" . $pk . "'";
+        if (is_array($pk)) {
+            //Avoid multiple choice error on klear
+            $this->_condition[] = "1=2";
```

Comment:
@Kaian I know, I just prefer it that way. It's kind of more universal. @cruzccl tiebreak this please :)